### PR TITLE
[chore] add kubernetes manifest

### PR DESCRIPTION
**Pull request by puckpuck**
Finishes what is needed for #545 

Adds the actual Kubernetes manifest after running `make generate-kubernetes-manifests`.

### CODE REVIEW

**Comment by puckpuck** [comment]
This is frustrating :( I thought #790 would add the proper files and configuration so the checks for this PR would pass. I'm now at a loss on how we get `yamllint` to ignore a specific file only.

**Comment by puckpuck** [comment]
Running `make yamllint` locally works without any error.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ release.
 
 * spanmetrics dashboard service&operation rates%latencies
 ([#787](https://github.com/open-telemetry/opentelemetry-demo/pull/787))
+* Adds Kubernetes manifests for the demo
+([#785](
```
Comment (fatsheep9146) [inline CHANGELOG.md:13]: the url is missing?

**Comment by austinlparker** [comment]
Can you try adding the path to the yamllint config? I wonder if it gets missed because of working directory shenanigans.

**Comment by julianocosta89** [comment]
merging #806 didn't solve 😭

**Comment by julianocosta89** [comment]
🥳🥳🥳

**Review by julianocosta89 — APPROVED** [review]
Should be good to go! 🤩

**Pull request merged**